tests/Microsoft.DotNet.Framework.Docker.Tests/ImageTests.cs

```diff
@@ -59,40 +59,66 @@ private static string GetFilterRegexPattern(string filter)
 
         [Theory]
         [MemberData(nameof(GetVerifyImagesData))]
-        public void VerifyImages(ImageDescriptor imageDescriptor)
+        public void VerifyImagesWithApps(ImageDescriptor imageDescriptor)
+        {
+            VerifyImages(imageDescriptor, "dotnetapp", true);
+        }
+
+        [Theory]
+        [MemberData(nameof(GetVerifyImagesData))]
+        public void VerifyImagesWithWebApps(ImageDescriptor imageDescriptor)
+        {
+            VerifyImages(imageDescriptor, "webapp", false);
+        }
+
+        private void VerifyImages(ImageDescriptor imageDescriptor, string appDescriptor, bool includeRuntime)
         {
             string baseBuildImage = $"{RepoOwner}/dotnet-framework-build:{imageDescriptor.BuildVersion}-{imageDescriptor.OsVariant}";
             VerifyImageExist(baseBuildImage);
 
-            string baseRuntimeImage = $"{RepoOwner}/dotnet-framework:{imageDescriptor.RuntimeVersion}-{imageDescriptor.OsVariant}";
-            VerifyImageExist(baseRuntimeImage);
+            string [] appBuildArgs;
+            if (includeRuntime)
+            {
+                string baseRuntimeImage = $"{RepoOwner}/dotnet-framework:{imageDescriptor.RuntimeVersion}-{imageDescriptor.OsVariant}";
+                VerifyImageExist(baseRuntimeImage);
+                appBuildArgs = new string[]
+                 {
+                        $"BASE_BUILD_IMAGE={baseBuildImage}",
+                        $"BASE_RUNTIME_IMAGE={baseRuntimeImage}",
+                 };
+            }
+            else
+            {
+                appBuildArgs = new string[]
+                 {
+                        $"BASE_BUILD_IMAGE={baseBuildImage}",
+                 };
+            }
 
-            string appId = $"dotnetapp-{DateTime.Now.ToFileTime()}";
+            string appId = $"{appDescriptor}-{DateTime.Now.ToFileTime()}";
             string workDir = Path.Combine(
                 Directory.GetCurrentDirectory(),
                 "projects",
-                $"dotnetapp-{imageDescriptor.RuntimeVersion}");
+                $"{appDescriptor}-{imageDescriptor.RuntimeVersion}");
 
             try
             {
                 DockerHelper.Build(
                     tag: appId,
                     dockerfile: Path.Combine(workDir, "Dockerfile"),
                     buildContextPath: workDir,
-                    buildArgs: new string[]
-                    {
-                        $"BASE_BUILD_IMAGE={baseBuildImage}",
-                        $"BASE_RUNTIME_IMAGE={baseRuntimeImage}",
-                    });
+                    buildArgs: appBuildArgs);
 
                 DockerHelper.Run(image: appId, containerName: appId);
             }
             finally
             {
                 DockerHelper.DeleteImage(appId);
             }
+
```

Comment:
There are a couple WS changes?  Were those intentional?

Reply:
I fixed a merge conflict, is this still an issue?

Reply:
yes it is.  You can see it very easily in the diff.

Reply:
What is WS? Sorry, I don't see it. All the changes in ImageTests.cs are intentional.

Reply:
WS is whitespace.  Your changes include new blank lines that I don't think add value and make the styling inconsistent.

Reply:
Got it, it will be removed.